migration_steps/prepare/create_skeleton_data/app/skeleton_data.py

```diff
@@ -1096,7 +1096,7 @@ def insert_finance_person_data_into_sirius(db_config, sirius_db_engine):
             id as person_id,
             row_number() over () + 990000 as finance_billing_reference,
             'DEMANDED'
-        from persons where clientsource = 'SKELETON';
+        from persons where clientsource = 'SKELETON' and id <> 520;
```

Comment:
What's the significance of 520?

Reply:
yeah it was just an existing record so he can recreate the error in local :)

Reply:
Worth commenting to avoid future confusion.

Reply:
Good call! I've added a comment, will merge with my next PR